apache-httpclient4/pom.xml

```diff
@@ -289,6 +294,7 @@
         <wiremock.version>2.5.1</wiremock.version>
         <httpcore.version>4.4.16</httpcore.version>
         <httpclient.version>4.5.14</httpclient.version>
```

Comment:
Remove this also

Reply:
closed this one as created new pr - https://github.com/eugenp/tutorials/pull/14293